engine/src/flutter/display_list/benchmarking/dl_benchmarks.cc

```diff
@@ -16,6 +17,26 @@
 namespace flutter {
 namespace testing {
 
+class DlPathVerbCounter : public DlPathReceiver {
```

Comment:
It occurs to me that I could have added a method to Dlpath for this. Hmm. It's only needed to report metrics for benchmarks though, so ... meh.